Automattic/wp-calypso
# Fixed an issue that caused an error when the theme list is emmpty

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4571

## Proposed Changes

* Guarded against an empty theme list when the event is triggered

## Testing Instructions



* Open the Calypso live link
* Go to the LiTS
* Filter the LiTS so that you only have one theme on the results list.
* Click that theme.
* There should be no console errors.
* The t.gif event `calypso_themeshowcase_theme_click` should contain an empty theme list.

No before and after images are provided since this is mainly a BE task.

## Pre-merge Checklist
